Client/Ecosia/UI/MyImpact/MyImpactCell.swift

```diff
@@ -202,7 +202,8 @@ final class MyImpactCell: UICollectionViewCell, AutoSizingCell, Themeable {
     }
     
     func update(personalCounter: Int) {
-        let progress = .init(personalCounter % 45) / 45.0
+        let searchesPerTree = TreeCounter.shared.statistics.searchesPerTree
```

Comment:
This looks like a great candidate for a new property on TreeCounter. Instead of calculating it everytime we need it.

Feel free to leave as is and we update on a upcoming PR.

Reply:
@vauxhall I'll give it some thoughts.. intuitively I'd rather add it as a derived getter to User.shared then.. as it connects closer to personalCounter value. What to you think?

Reply:
Sounds good 👍
a computed property similar to `searchImpact` would be handy.